src/components/Dialogs/Dialogs.jsx

```diff
@@ -14,24 +15,52 @@ const DialogItem = props => {
   );
 };
 
-const Message = props => {
+const Message = (props) => {
   return <div className={s.dialogs}>{props.message}</div>;
 };
 
-const dialogsElements = (usersData) => usersData.map(dialog => (
-  <DialogItem name={dialog.name} id={dialog.id} />
-));
+const dialogsElements = (usersData) =>
+  usersData.map((dialog) => <DialogItem name={dialog.name} id={dialog.id} />);
 
-const messagesElements = (messagesData) => messagesData.map(message => (
-  <Message message={message.message} />
-));
+const messagesElements = (messagesData) =>
+  messagesData.map((message) => <Message message={message.message} />);
 
 const Dialogs = (props) => {
+  let newMessageBody = props.newMessageBody;
+
+  let onSendMessageClick = () => {
```

Comment:
Consider using "const" instead of "let" if a variable is not reassigned in the current scope (current scope = Dialogs function)

Usually, function expressions are not reassigned, so you can default tot using "const" with them.

Reply:
done